core/commons/CommonsManager.py

```diff
@@ -82,10 +78,13 @@ def payload(message: MQTTMessage) -> dict:
 			p = p.decode()
 
 			if p == 'true' or p == 'false':
+				# TODO this is the same as return {str(p): p}, but less readable
+				# in this case even return {p: p} should be enough since it is already a string
 				raise ValueError
 
```

Comment:
Problem here are payloads differently structured, like payloads that are not key value based but only send a string per exemple